civihr_default_theme/scss/pages/_features-in-civihr.scss

```diff
@@ -0,0 +1,172 @@
+$carousel-box-shadow-color: rgba(#3f3c55, 0.3);
+
+.page-features-in-civihr {
+  .onboarding_wizard_features_get_started {
+    background: $crm-wizard-secondary-color;
+    display: block;
+    margin: 40px auto 0;
+    text-transform: uppercase;
+    width: 200px;
+  }
+
+  .page-header {
+    font-weight: 700;
+    margin-top: 80px;
+    text-align: center;
+
+    .title {
+      font-weight: 500;
```

Comment:
Right now there is no variable for font-weight 500. So the same should be created and used here.